tests/python/gpu/test_operator_gpu.py

```diff
@@ -49,25 +49,57 @@ def test_convolution_with_type():
     check_consistency(sym, ctx_list)
 
 def test_convolution_options():
+    ctx_list = [{'ctx': mx.gpu(0), 'conv_data': (2, 2, 7), 'type_dict': {'conv_data': np.float64}},
+                {'ctx': mx.gpu(0), 'conv_data': (2, 2, 7), 'type_dict': {'conv_data': np.float32}},
+                {'ctx': mx.gpu(0), 'conv_data': (2, 2, 7), 'type_dict': {'conv_data': np.float16}},
+                {'ctx': mx.cpu(0), 'conv_data': (2, 2, 7), 'type_dict': {'conv_data': np.float64}},
+                {'ctx': mx.cpu(0), 'conv_data': (2, 2, 7), 'type_dict': {'conv_data': np.float32}}]
+    # 1D convolution
+    sym = mx.sym.Convolution(num_filter=3, kernel=(3,), pad=(1,), name='conv')
```

Comment:
add consistency check agains convolution_v1. You can use a list of symbol for sym to run different sym for each context